tests/test_clean.py

```diff
@@ -710,20 +740,6 @@ def test_svg_allow_local_href_nonlocal(text, expected):
     assert clean(text, tags=TAGS, attributes=ATTRS) == expected
 
 
-def test_weird_strings():
-    s = '</3'
-    assert clean(s) == ''
-
-
-@pytest.mark.xfail(reason='regression from bleach 1.5')
-def test_sarcasm():
-    """Jokes should crash.<sarcasm/>"""
-    assert (
-        clean('Yeah right <sarcasm/>') ==
-        'Yeah right &lt;sarcasm/&gt;'
-    )
-
-
```

Comment:
I merged a bunch of tests into a big `stripping` test with many test cases and a big `escaping` test with many test cases. It cleaned up this file a bunch plus added some additional coverage.